addon/base-record-array.js

```diff
@@ -79,9 +80,9 @@ if (CUSTOM_MODEL_CLASS) {
     static create(...args) {
       let instance = super.create(...args);
 
-      let proxy = new Proxy(instance, baseRecordArrayProxyHandler);
-      MANAGED_ARRAYS.add(proxy);
-      return proxy;
+      let arr = [];
+      arr._instance = instance;
```

Comment:
can we rename `__instance` to be less cryptic? it's the record array right?  wouldn't `__recordArray` be more clear?

Reply:
Done